src/managers/InMemoryTaskManager.java

```
@@ -98,6 +128,10 @@ public void printSubtask() {
             for (Integer id : subtaskList.keySet()) {
```

Лучше стараться, чтобы код не уходил слишком сильно в право. Пример, if (!subtaskList.isEmpty()) { можно не делать здесь. Можно сделать if (subtaskList.isEmpty()) {System.out.println("Список подзадач пуст"); return. А после этого уже сделать код который написан выше. Это называется быстрый выход (fail fast) из метода. Но задача писать код без сильной сдвижки вправо.

---

done